<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Konversi</string>
  </property>
  <widget class="QPushButton" name="apus">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>153</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka2">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>182</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka3">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>182</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka7">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>238</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka5">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>211</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka8">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>209</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="spasi">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>151</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="AC1">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>153</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AC</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka6">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>211</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka1">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Handwriting</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>KONVERSI MATA UANG</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="line2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="titik">
   <property name="geometry">
    <rect>
     <x>184</x>
     <y>271</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="angka9">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pilihan1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="pilihan2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="angka0">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 123, 163)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>apus</zorder>
  <zorder>angka2</zorder>
  <zorder>angka3</zorder>
  <zorder>angka7</zorder>
  <zorder>angka5</zorder>
  <zorder>angka8</zorder>
  <zorder>angka4</zorder>
  <zorder>spasi</zorder>
  <zorder>AC1</zorder>
  <zorder>angka6</zorder>
  <zorder>angka1</zorder>
  <zorder>label</zorder>
  <zorder>line2</zorder>
  <zorder>line1</zorder>
  <zorder>titik</zorder>
  <zorder>angka9</zorder>
  <zorder>pilihan1</zorder>
  <zorder>pilihan2</zorder>
  <zorder>angka0</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
